pkg/solace/messaging_service.go

```diff
@@ -121,6 +121,29 @@ type MessagingService interface {
 
 	// Info returns the API Info for this MessagingService instance.
 	Info() metrics.APIInfo
+
+        // Updates the value of a modifiable service property once the service has been created.
+        // Modifiable service properties include:
+        //     - solace/config.*AuthenticationPropertySchemeOAuth2AccessToken,
```

Comment:
No need for * as the properties are constants.

Reply:
I removed the asterisks for the constants